pytest.ini

```diff
@@ -0,0 +1,9 @@
+[pytest]
+python_files = *.py
+norecursedirs = .* _* node node_modules coverage venv
```

Comment:
I'm thinking about restructuring the tests here (and eventually in the template) to be more normal, and more easily excluded from the Django code tree. Like:

```
┬ metashare
│ └ foo
│    └ bar.py
└ tests
  └ foo
     └ test_bar.py
```

Thoughts?

Reply:
Any particular motivating reason? I haven't had trouble with the current layout so I'm not sure it's worth the effort to change.

Reply:
Two reasons: a change in my philosophy on the question (which would not be worth changing here) and the fact that tests for templatetags need special-casing because of how Django detects available templatetags.